modules/districtmDMXBidAdapter.js

```diff
@@ -310,4 +375,65 @@ export function bindUserId(eids, value, source, atype) {
     })
   }
 }
+
+export function getApi({protocols}) {
+  let defaultValue = [2];
+  let listProtocols = [
+    {key: 'VPAID_1_0', value: 1},
+    {key: 'VPAID_2_0', value: 2},
+    {key: 'MRAID_1', value: 3},
+    {key: 'ORMMA', value: 4},
+    {key: 'MRAID_2', value: 5},
+    {key: 'MRAID_3', value: 6},
+  ];
+  if (protocols) {
+    return listProtocols.filter(p => {
+      return protocols.includes(p.key);
```

Comment:
tests fail on ie 11 bc https://caniuse.com/#search=includes

Reply:
includes was changed for indexOf !== -1